_data/people/Dozzllam.yml

```diff
@@ -0,0 +1,7 @@
+username: Dozzllam
+
+name:paulus shituna
```

Comment:
I think you'll need a space here between the key and value.

Try:

`name: paulus shituna`